Examples/UIComponentExample/Examples/Complex layout/ComplexLayoutViewController.swift

```diff
@@ -65,13 +74,35 @@ class ComplexLayoutViewController: ComponentViewController {
               }
             }
           }
-        }.inset(h: 20).scrollView().showsHorizontalScrollIndicator(false).animator(AnimatedReloadAnimator())
+        }.inset(h: 20).inset(top: 10).scrollView().showsHorizontalScrollIndicator(false).animator(AnimatedReloadAnimator())
```

Comment:
No animation.
How to do in order to have animation?

Reply:
you are using the `.animator` modifier instead of using the automatic dynamicMemberLookup update modifier.
You can use `.with(\.animator, AnimatedReloadAnimator())` for this.